components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/restapi/publisher/SearchApiServiceImplUtil.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This has to WSO2 LLC. Check other places as well.

Reply:
Resolved in a05bcd7fc00248a0d25d1496cf127e5caee4cdfc